src/apis/domains/host/usePatchHostInfo.ts

```diff
@@ -0,0 +1,52 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { useNavigate } from 'react-router-dom';
+
+import { patch } from '@apis/api';
+import { QUERY_KEY } from '@apis/queryKeys/queryKeys';
+
+import { components } from '@schema';
+import { ErrorResponse, ErrorType, MutateResponseType } from '@types';
+
+type HostUpdateRequest = components['schemas']['HostUpdateRequest'];
+export interface PatchHostInfoRequest {
+  hostId: number;
+  hostInfoValue: HostUpdateRequest;
+}
+
+const patchHostInfo = async ({
+  hostId,
+  hostInfoValue,
+}: PatchHostInfoRequest): Promise<MutateResponseType> => {
+  try {
+    const response = await patch<MutateResponseType>(`/v2/host/${hostId}`, hostInfoValue);
```

Comment:
```suggestion
    const response = await patch<ApiResponseType<MutateResponseType>>(`/v2/host/${hostId}`, hostInfoValue);
```
저희 schema 이용 컨벤션처럼 patch type을 바꿔야합니다!

Reply:
그럼 response도 아래처럼 바뀌어야 하겠죠?
```tsx
return response.data.data
```
왜 data를 한 번 더 받아야하는지, ApiResponseType 들어가셔서 확인해보시면 기초 제네릭을 이해하는데에 도움이 많이 될겁니다 ㅎㅎ

Reply:
오호,, 꼼꼼한 리뷰 감사합니다! 반영했습니다 😋